src/core/index.js

```diff
@@ -275,6 +266,40 @@ util.extend( corefn, {
     return this._private.container;
   },
 
+  mount: function( container, rendererOptions ){
+    if( container == null ){ return; }
+
+    let cy = this;
+    let _p = cy._private;
+    let options = _p.options;
+    
+    let rOpts = rendererOptions ? rendererOptions : { name: 'canvas' };
```

Comment:
Does it make sense to ignore previous renderer options for both cases (when rendererOptions is supplied and when  it is not? )

Reply:
I think so.  Any internal call to `mount()` that wanted the previous options taken into account would pass the previous options in, I think.